.github/CODEOWNERS

```diff
@@ -1,3 +1,3 @@
 # Add a team or username to this file
 # Example:
-# *   @ministryofjustice/operations-engineering
+# * @ministryofjustice/analytical-platform
```

Comment:
Should this be uncommented?

Reply:
Nah this is a template, it could remain "operations-engineering", but I just ported code from Python template